src/components/features/PairwiseSearch/InteractorSearch.vue

```diff
@@ -71,7 +71,7 @@
         </div>
       </div>
       <v-card-text class="interactingPathwaysCard">
-        <v-card outlined>
+        <v-card v-if="isCytoscapeView" outlined>
```

Comment:
Can you check: In the pathway plot view, search for another protein and see if the view can be switched to network or stay at the pathway plot? If at the pathway plot, how slow is it? Note: this is a progress pane showing the loading.